manifests/init.pp

```diff
@@ -0,0 +1,29 @@
+# Installs and configures Puppet Bolt for use within the SIMP enviroment
+#
+# @param bolt_server
+#   If true, will install and configure the Puppet Bolt package
+#
+# @param package_name
+#   The name of the Puppet Bolt rpm package
+#
+# @author SIMP Team <https://simp-project.com/>
+# 
+class simp_bolt (
+  Boolean  $bolt_server  = false,
+  String   $package_name = 'puppet-bolt'
+) {
+
+  simplib::assert_metadata($module_name)
+
+  include '::simp_bolt::user'
```

Comment:
I wouldn't do this by default.

I would probably expect this class to set up the bolt user directories but I would not expect it to place a local user onto all of my systems since this requires justification.

You may just want to split up the user creation portion from the files and directories so that reference material can be added but without adding a user to the system unless the users want one.

The instructions should be clear that you need some type of user to be active on nodes that bolt will be controlling.

Splitting this into `bolt::controller` and `bolt::target` *might* make it more clear. Not quite sure.

Reply:
So would you suggest classes controller and target and moving user under target with an option to manage the account and install and config under controller?  Also include something optional in controller to specify and manage controller user?

Reply:
Reconfigured module to create controller and target classes.  Target class is mandatory because it specifies the user account on remote systems, which the portions of the controller class rely upon, but creating the account and locking it down are now configured as options set by boolean parameters in target.